ui/tests/acceptance/secrets/backend/kubernetes/overview-test.js

```diff
@@ -11,7 +11,8 @@ import kubernetesHandlers from 'vault/mirage/handlers/kubernetes';
 import authPage from 'vault/tests/pages/auth';
 import { visit, click, currentRouteName } from '@ember/test-helpers';
 import { selectChoose } from 'ember-power-select/test-support';
-import { SELECTORS } from 'vault/tests/helpers/kubernetes/overview';
+import { KUBERNETES_OVERVIEW } from 'vault/tests/helpers/components/kubernetes/page/overview-selectors';
```

Comment:
Reviewing the pattern notes,  it sounded like we landed on all selectors (including page) being in the resource directory, and not `components/`?

> Combine test selectors into one file
> - Example of path: helpers/clients/client-count-selectors.js
> - Components first and then page specific selectors, separated by comment


```suggestion
import { KUBERNETES_OVERVIEW } from 'vault/tests/helpers/kubernetes/kubernetes-selectors';
```